src/schema/schema_open.c

```diff
@@ -720,6 +720,11 @@ __wt_schema_open_layered(WT_SESSION_IMPL *session)
     WT_LAYERED_TABLE *layered;
     uint32_t ingest_id, stable_id;
 
+    if (!__wt_conn_is_disagg(session)) {
+        __wt_err(session, EINVAL, "layered table is only supported for disaggregated storage");
```

Comment:
I'd love if this wasn't the case. Is there a fundamental reason why layered tables need disaggregated storage? Perhaps this is more that I would prefer WiredTiger didn't have something that is called a "disaggregated storage" mode, but I think it's important that we can test this as broadly as possible.

Is there a way that layered tables could be backed by two local tables?

Reply:
I understand the concern. It can be backed by two local tables potentially but we are not supporting it now. If we want to do that, we can certainly change this.

Reply:
Currently, if you create a layered table without setting the page_log extension, you will get a crash. So I think it is better to limit when you can create a layered table now instead of crashing.